### PR TITLE
Task sa investigation child routers

### PR DESCRIPTION
**Pull request by SRAlexander**

### CODE REVIEW

File: app/src/types/blocks/lloydGeorgeActions.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { REPOSITORY_ROLE } from '../generic/authRole';
-import { routes } from '../generic/routes';
+import { routeChildren } from '../generic/routes';
 import { LG_RECORD_STAGE } from './lloydGeorgeStages';
```
Comment (RioKnightleyNHS) [inline app/src/types/blocks/lloydGeorgeActions.ts:4]: type ActionRoute = routeChildren && routes

File: app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx [hunk 2]
```diff
@@ -156,58 +139,75 @@ function LloydGeorgeDownloadAllStage({
...
-    return inProgress ? (
-        <div className="lloydgeorge_downloadall-stage" data-testid="lloydgeorge_downloadall-stage">
-            <div className="lloydgeorge_downloadall-stage_header">
-                <h1>{pageHeader}</h1>
-                <h2>{patientDetails?.givenName + ' ' + patientDetails?.familyName}</h2>
-                <h4>NHS number: {patientDetails?.nhsNumber}</h4>
-                <div className="nhsuk-heading-xl" />
-                <h4>Preparing download for {numberOfFiles} files</h4>
-            </div>
+    return (
```
Comment (RioKnightleyNHS) [inline app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx:142]: const Index = () => (<div> ... </div>)

File: app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx [hunk 3]
```diff
@@ -156,58 +139,75 @@ function LloydGeorgeDownloadAllStage({
...
+        <>
+            <Routes>
+                <Route
+                    index
+                    element={
+                        <div
```
Comment (RioKnightleyNHS) [inline app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx:148]: ```element={<Index/>}```

File: app/src/pages/documentSearchResultsPage/DocumentSearchResultsPage.tsx [hunk 4]
```diff
@@ -78,71 +79,94 @@ function DocumentSearchResultsPage() {
...
+    return (
         <>
-            <h1 id="download-page-title">{pageHeader}</h1>
+            <div>
+                <Routes>
+                    <Route
+                        index
+                        element={
+                            <>
+                                <h1 id="download-page-title">{pageHeader}</h1>
```
Comment (RioKnightleyNHS) [inline app/src/pages/documentSearchResultsPage/DocumentSearchResultsPage.tsx:91]: Same as above with an Index component

File: app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx [hunk 5]
```diff
@@ -196,6 +205,7 @@ function LloydGeorgeUploadPage() {
             const uploadingDocuments = documents.map((doc) => {
                 const documentMetadata = uploadSession[doc.file.name];
                 const documentReference = documentMetadata.fields.key;
+                // const documentReference = "test123"
```
Comment (RioKnightleyNHS) [inline app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx:208]: remove comment

**Review by RioKnightleyNHS — CHANGES_REQUESTED** [review]

File: app/src/types/blocks/lloydGeorgeActions.ts (hunk 1 above)
Comment (SRAlexander) [reply app/src/types/blocks/lloydGeorgeActions.ts:4]: Modified

File: app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx (hunk 2 above)
Comment (SRAlexander) [reply app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx:142]: Modified

File: app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx (hunk 3 above)
Comment (SRAlexander) [reply app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadAllStage/LloydGeorgeDownloadAllStage.tsx:148]: Modified

File: app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx (hunk 5 above)
Comment (SRAlexander) [reply app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx]: Done

File: app/src/pages/documentSearchResultsPage/DocumentSearchResultsPage.tsx (hunk 4 above)
Comment (SRAlexander) [reply app/src/pages/documentSearchResultsPage/DocumentSearchResultsPage.tsx]: done

**Review by joefong-nhs — APPROVED** [review]

**Pull request merged**